Revit_Core_Engine/Query/ColumnLine.cs

```diff
@@ -0,0 +1,73 @@
+/*
+ * This file is part of the Buildings and Habitats object Model (BHoM)
+ * Copyright (c) 2015 - 2022, the respective contributors. All rights reserved.
+ *
+ * Each contributor holds copyright over their respective contributions.
+ * The project versioning (Git) records all such contribution source information.
+ *                                           
+ *                                                                              
+ * The BHoM is free software: you can redistribute it and/or modify         
+ * it under the terms of the GNU Lesser General Public License as published by  
+ * the Free Software Foundation, either version 3.0 of the License, or          
+ * (at your option) any later version.                                          
+ *                                                                              
+ * The BHoM is distributed in the hope that it will be useful,              
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+using BH.Engine.Geometry;
+using BH.oM.Base.Attributes;
+using BH.oM.Geometry;
+using BH.oM.Physical.Elements;
+using System;
+using System.ComponentModel;
+
+namespace BH.Revit.Engine.Core
+{
+    public static partial class Query
+    {
+        /***************************************************/
+        /****              Public methods               ****/
+        /***************************************************/
+
+        [Description("Extracts the location line of a BHoM Column object.")]
+        [Input("column", "A BHoM Column object.")]
+        [Output("line", "The location line of the BHoM Column object.")]
+        public static Line ColumnLine(this Column column)
+        {
+            if (column == null)
+            {
+                BH.Engine.Base.Compute.RecordError(string.Format("Cannot read location line because column is null. BHoM_Guid: {0}", column.BHoM_Guid));
```

Comment:
These days I personally I prefer string formatting based on `$""` syntax instead of the good old `string.Format`. Please feel free to stick to the current version if this is your preference, but if not then I would recommend using the more contemporary syntax - it makes the code a bit more readable.

Just in case, this is a general comment to all code that you will be writing, but again, please be aware that this is just my preference and we are all allowed to have our own 😉

Reply:
I would second the use of  `$""`! :smile:

Reply:
Personally I was considering a dedicated push to replace all `String.Format` with `$""` across the entire BIM-related code base...

Reply:
That wouldn't be such a bad idea for consistency @pawelbaran - could be a nice easy compliance task for someone to learn more about the code base... 😉

Reply:
Man, I would write a script for it 😃

Reply:
Just be careful not to fall foul of the automated code modification regulations set out in our BHoMBot procedures then 😉

Reply:
Did someone say foul?

![image](https://media2.giphy.com/media/4m7zfhchURMRO/giphy.gif)

Reply:
Nothing is fouler than the code contained in any BIM related repository...

![image](https://media.giphy.com/media/l2JejeFKeTHE8mVFu/giphy.gif)